src/main/java/ch/hearc/todont/TestController.java

```diff
@@ -0,0 +1,23 @@
+package ch.hearc.todont;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class TestController {
```

Comment:
We should probably delete this controller before merging so it does not conflict with the real controllers